originations/services/pubsub/async_publisher.py

```diff
@@ -4,6 +4,14 @@
 from google.cloud import pubsub_v1
 from originations.config.config import settings
 from originations.services.logging import log_handler
+from typing import Any, Optional
+
+
+def nullable(type: str, value: Optional[Any] = None) -> Optional[dict]:
+    if not value:
+        return value
+    else:
+        return {type: value}
```

Comment:
Said helper function. Either gives null when there is no data or {type : value} when there is data. 

This is tested and fully compatible with BigQuery